ramls/userSummary.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "User Summary object allows to check each of the block conditions for a patron",
+  "properties": {
+    "id": {
+      "description": "ID of the patron for which all conditions are stored",
+      "type": "string",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+    },
+    "outstandingFeeFineBalance": {
+      "description": "Outstanding fee/fine balance",
+      "type": "number",
+      "javaType" : "java.math.BigDecimal"
+    },
+    "numberOfOpenFeesFinesForLostItems": {
+      "description": "Number of open fees/fines for lost items",
+      "type": "integer"
+    },
+    "numberOfLostItems": {
+      "description": "Number of lost items",
+      "type": "integer"
+    },
+    "openLoans": {
+      "type": "array",
+      "items": {
+        "description": "Collection of open loans",
+        "type": "object",
+        "properties": {
+          "loanId": {
+            "description": "Loan id",
```

Comment:
Let's make "ID" upper-case in human-readable text (everywhere, not just in this dexcription)